concepts/05 Widgets/Form/01 Getting Started with Form/80 Submit the Form.md

```diff
@@ -0,0 +1,282 @@
+To submit a form, add the [Button Item](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/ButtonItem/) and set the [useSubmitBehavior](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#useSubmitBehavior) property to **true**. The Form can be submitted to a server only when all the validation checks pass succefully. 
+
+In this tutorial, we use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setTimeout" target="_blank">setTimeout</a> function to emulate the form submission. The Form is wrapped in the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" target="_blank">form</a> element. We set the [preventDefault](/Documentation/ApiReference/Common/Object_Structures/dxEvent/Methods/#preventDefault) property to **true** to override the HTML form submit event:
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#form").dxForm({
+            formData: {
+                // ...
+            },
+            colCount: 2,
+            items: [{
+                itemType: "group",
+                caption: "Personal Information",
+                colCount: 2,
+                items: [{
+                    dataField: "name",
+                    isRequired: "true"
+                }, {
+                    dataField: "officeNumber",
+                    validationRules: [{
+                        type: "numeric",
+                        message: "This field should contain a number"
+                    }] 
+                }, {
+                    dataField: "email",
+                    validationRules: [{
+                        type: "email",
+                        message: "This is not a valid Email"
+                    }] 
```

Comment:
Keep your samples succinct and to the point. There's no need to show full items configuration.